tests/integration/targets/ec2_transit_gateway/tasks/main.yml

```diff
@@ -10,159 +10,168 @@
       region: '{{ aws_region }}'
   block:
 
-    - name: generate unique value for testing
+    - name: Generate unique value for testing
       set_fact:
         tgw_description: "{{ resource_prefix }}-tgw"
 
-    - name: test create transit gateway without tags
```

Comment:
Can you also please ensure that check_mode for all the functionalities is tested?

Reply:
added more tests